### PR TITLE
[EHL] Fix S4 issue in Windows

### PR DESCRIPTION
**Pull request by jinjhuli**
Fixes an issue where S4 is not working in
Windows when a PCIE card is attached.

Signed-off-by: jinjhuli <jin.jhu.lim@intel.com>

### CODE REVIEW

**Comment by ywho79** [comment]
We do not need those SDBG functions in ACPI table

**Comment by jinjhuli** [comment]
> We do not need those SDBG functions in ACPI table

Thanks for the review, I've removed it.

File: Platform/ElkhartlakeBoardPkg/AcpiTables/Dsdt/PlatformNvs.asl [hunk 1]
```diff
@@ -1552,62 +1557,119 @@
...
+  //
+  Offset(2503),   ECLE, 8,  // Offset(2503),  PSE Eclite Service Enable/Disable
+  Offset(2504),   PWOL, 8,  // Offset(2504),  PSE WoL Service Enable/Disable
+  #ifdef RESTRICTED_FLAG
+  // Below is InternalOnly NVS region
+  Offset(2505),   EPIE, 8,  // Offset(2505),  Enable EPI Device
+  Offset(2506),   EPGI, 32, // Offset(2506),  EPI Device Interrupt
+  Offset(2510),   DWFK, 32, // Offset(2510),  Gpio for WiFi RF-Kill
+  Offset(2514),   SRLD, 8,  // Offset(2514),  Serial Port ACPI debug
+  #endif
```
Comment (myguitar) [inline Platform/ElkhartlakeBoardPkg/AcpiTables/Dsdt/PlatformNvs.asl:1674]: Please remove.

File: Silicon/ElkhartlakePkg/Include/PlatformNvsAreaDef.h [hunk 2]
```diff
@@ -1328,64 +1338,121 @@ typedef struct {
...
+  //
+  UINT8    EcliteServiceEnable;                     ///< Offset 2503    PSE Eclite Service Enable/Disable
+  UINT8    PseWolServiceEnable;                     ///< Offset 2504    PSE WoL Service Enable/Disable
+  #ifdef RESTRICTED_FLAG
+  // Below is InternalOnly NVS region
+  UINT8    EpiDeviceEnable;                         ///< Offset 2505    Enable EPI Device
+  UINT32   EpiGpioInterrupt;                        ///< Offset 2506    EPI Device Interrupt
+  UINT32   DiscreteWiFiRfKillGpio;                  ///< Offset 2510    Gpio for WiFi RF-Kill
+  UINT8    SerialPortAcpiDebug;                     ///< Offset 2514    Serial Port ACPI debug
+  #endif
```
Comment (myguitar) [inline Silicon/ElkhartlakePkg/Include/PlatformNvsAreaDef.h:1455]: Please remove.

Comment (jinjhuli) [reply Silicon/ElkhartlakePkg/Include/PlatformNvsAreaDef.h]: Done, thanks @myguitar for the review

File: Platform/ElkhartlakeBoardPkg/AcpiTables/Dsdt/PlatformNvs.asl (hunk 1 above)
Comment (jinjhuli) [reply Platform/ElkhartlakeBoardPkg/AcpiTables/Dsdt/PlatformNvs.asl]: Done, thanks @myguitar for the review

File: Platform/ElkhartlakeBoardPkg/AcpiTables/SsdtRtd3/EhlCrbRtd3.asl [hunk 3]
```diff
@@ -116,44 +119,9 @@ External(WWAN_PCIE_ROOT_PORT.LASX)
...
-      Store(SDR1, Index(RSTG, 1))
-      Name(PWRG, Package() {0, 0})
-      Store(SD2P, Index(PWRG, 0))
-      Store(SDP1, Index(PWRG, 1))
-      Name(WAKG, 0)             // must be defined due to compiler bug, will be removed when fixed https://bugs.acpica.org/show_bug.cgi?id=1432
-      Name(WAKP, 0)             // must be defined due to compiler bug, will be removed when fixed https://bugs.acpica.org/show_bug.cgi?id=1432
-      Name(SCLK, 0)
-      Include("Rtd3Pcie.asl")
-      Scope(\_SB.PC00.RP13.PXSX) {
+      Scope(\_SB.PC00.RP01.PXSX) {
```
Comment (myguitar) [inline Platform/ElkhartlakeBoardPkg/AcpiTables/SsdtRtd3/EhlCrbRtd3.asl:124]: RP09 and RP13 do not require RTD asl?

Comment (jinjhuli) [reply Platform/ElkhartlakeBoardPkg/AcpiTables/SsdtRtd3/EhlCrbRtd3.asl:124]: Yes, it is not require and BIOS doesn't have these too.

**Review by myguitar — APPROVED** [review]

**Review by gdong1 — APPROVED** [review]

**Pull request merged**